### PR TITLE
Refactor master branch for ES6

### PR DESCRIPTION
**Pull request by nrebhun**
## Summary:
- handledata.js
    - refactor of master branch to ES6
    - removal of unnecessary _Observer_ class
    - removal of Begin class comments

- package.json
    - added script for running: `$ npm run build` (target is **lib/**)
    - added _ES2015_ dev dependency
    - updated _BabelJS_ dev dependency

- .gitignore
    - added line to ignore **lib/**

- .babelrc
    - created Babel config file for _ES2015_ preset use

### CODE REVIEW

File: handleData.js [hunk 1]
```diff
@@ -9,101 +9,108 @@ function processCharacterData(dataSetSize) {
...
+                this.timeStamp.getDate() + "/" +
+                (this.timeStamp.getMonth() + 1) + "/" +
+                this.timeStamp.getFullYear() + " @ " +
+                this.timeStamp.getHours() + ":" +
+                this.timeStamp.getMinutes() + ":" +
+                this.timeStamp.getSeconds();
+    }
 }
 
 // Begin MessageThread Definition
```
Comment (zmbush) [inline handleData.js]: This comment seems pretty unnecessary now.

File: handleData.js [hunk 2]
```diff
@@ -9,101 +9,108 @@ function processCharacterData(dataSetSize) {
...
+        this.dungeonMaster.addressBook = this.dungeonMaster.addressBook.concat(this.characters);
+        for (var i = 0; i < this.characters.length; i++) {
+            this.characters[i].addressBook.push(this.dungeonMaster);
+            this.characters[i].addressBook = this.characters[i].addressBook.concat(this.characters);
+            this.characters[i].addressBook.splice(i+1, 1);
+        }
     }
 }
 
 // Begin Player Definition
```
Comment (zmbush) [inline handleData.js]: This one too

File: handleData.js (hunk 1 above)
Comment (nrebhun) [inline handleData.js]: Yeah I think most of those comments are, at this point. 
Since they are nicely bundled together in "classes"

**Comment by zmbush** [comment]
This LGTM :+1:

**Comment by nrebhun** [comment]
Okay, one last check before the merge?
And jibbers crabst, I can see where this is going to get messy, when merging the feat/* branches into master going forward. I'm glad this is happening now, rather than later.

**Comment by zmbush** [comment]
LGTM

**Pull request merged**